kasa/iot/modules/light.py

```diff
@@ -30,7 +30,7 @@ def _initialize_features(self):
         super()._initialize_features()
         device = self._device
 
-        if self._device.is_dimmable:
+        if self._device._is_dimmable:
```

Comment:
Should we avoid using these, now-deprecated(?) methods, or is the plan to keep these alive as internal helpers also in the future?

Reply:
So this is a result of my shortcut for the `iot` `Light` module to just be a wrapper around the `IotBulb` and `IotDimmer` devices.  The proper way to do this would be to move the actual code into the Iot module but I think we should consider that for a later release to reduce the churn on the `iot` code.